### PR TITLE
Improved and create new chaos examples

### PR DESCRIPTION
**Pull request by mbaudin47**
This PR improves the polynomial chaos and S.A. examples. 

The order of the examples in the "Metamodeling" section is changed so that simpler examples come first. 

* chaos_ishigami.ipynb: this example should come first, given that it is classical, easy and simple.
* chaos_cantilever_beam_integration.ipynb: *new*. It shows the integration method, which had no example before. **Please check** that the polynomial degree selected in `FixedStrategy` is consistent with respect to the parameters of the `GaussProductExperiment` class. 
* functional_chaos.ipynb: fixed. The example was wrong in the sense that the metamodel validation was so poor that the fact that the predictivity was close to zero could not be seen. To simplify the task, I changed the model a bit so that high polynomial degrees are not required anymore. This is still a difficult example, nonlinear with high interactions. 
* functional_chaos_advanced_ctors.ipynb: beautified by printing the comments in the text.
* functional_chaos_exploitation.ipynb
* functional_chaos_database.ipynb
* chaos_ishigami_grouped_indices.ipynb
* chaos_build_distribution.ipynb: *new* - the role of this function was not clear enough from the existing doc.
* functional_chaos_graphs.ipynb: simplified
* chaos_draw_validation.ipynb: separated from functional_chaos_graphs.ipynb and updated to use the dedicated MetamodelValidation class.

Also clarified the "Reliability, Sensitivity" section.
* Separated Reliability from Sensitivity examples in the table of contents.
* Clarified the example which shows the S.A. by sampling for the Ishigami test case in sensitivity_sobol.ipynb. Clarified how to estimate first and total indices (without second order indices) and including second order indices. Showed how to change the estimator.

### CODE REVIEW

**Comment by mbaudin47** [comment]
This only partially solves #1214: this is because the current example in chaos_cantilever_beam_integration.ipynb does not show how to use MonteCarlo, LHS or low discrepancy experiments in the integration method.

File: python/doc/examples/meta_modeling/functional_chaos.ipynb [hunk 1]
```diff
@@ -11,11 +11,21 @@
...
    "metadata": {},
    "source": [
-    "In this basic example we are going to create a global approximation of a model response using functional chaos.\n",
+    "In this example we are going to create a global approximation of a model response using functional chaos.\n",
     "\n",
-    "Here\n",
+    "Let $h$ be the function defined by:\n",
+    "$$\n",
+    "g(\\mathbf{x}) = \\left[\\cos(x_1 + x_2), (x_2 + 1) e^{x_1}\\right]\n",
+    "$$\n",
```
Comment (jschueller) [inline python/doc/examples/meta_modeling/functional_chaos.ipynb]: In this PR too, can you make sure you add a blank line after each closing $$ ?

Comment (mbaudin47) [reply python/doc/examples/meta_modeling/functional_chaos.ipynb]: Oups. Fixed.

**Comment by jschueller** [comment]
see #1296

**Pull request closed without merging**